src/python/pants/notes/1.30.x.rst

```diff
@@ -0,0 +1,205 @@
+1.30.0dev0 (6/05/2020)
+----------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Remove deprecated v1 functionality (#9996)
+  `PR #9996 <https://github.com/pantsbuild/pants/pull/9996>`_
+
+* Rename ReportType to CoverageReportType and move it to test.py (#9991)
+  `Issue #9968
 <https://github.com/pantsbuild/pants/issues/9968
>`_
+  `PR #9991 <https://github.com/pantsbuild/pants/pull/9991>`_
+
+* Remove the passthrough-arguments-with-multiple-goals deprecation. (#9984)
+  `PR #9984 <https://github.com/pantsbuild/pants/pull/9984>`_
+
+* Remove `--files-not-found-behavior=ignore` (#9986)
+  `PR #9986 <https://github.com/pantsbuild/pants/pull/9986>`_
+
+* Change --filedeps-absolute option to default to False. (#9983)
+  `PR #9983 <https://github.com/pantsbuild/pants/pull/9983>`_
+
+* Custom target types now need Target API bindings for full support (#9981)
+  `PR #9981 <https://github.com/pantsbuild/pants/pull/9981>`_
+
+* Remove deprecated source root trie. (#9936)
+  `PR #9936 <https://github.com/pantsbuild/pants/pull/9936>`_
+
+* Allow @rules to set their levels, and default to debug (#9940)
+  `PR #9940 <https://github.com/pantsbuild/pants/pull/9940>`_
+
+* Enable pantsd by default (#9935)
+  `PR #9935 <https://github.com/pantsbuild/pants/pull/9935>`_
+
+* Remove codeanalysis (Kythe) contrib plugin due to low usage (#9918)
+  `PR #9918 <https://github.com/pantsbuild/pants/pull/9918>`_
+
+* Remove AWS Lambda contrib plugin in favor of V2 backend (#9917)
+  `PR #9917 <https://github.com/pantsbuild/pants/pull/9917>`_
+
+* Remove V1 isort and `lint.pythonstyle` in favor of V2 isort and Flake8 (#9909)
+  `PR #9909 <https://github.com/pantsbuild/pants/pull/9909>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Create FFI functions for handling digests on workunits (#9965)
+  `PR #9965 <https://github.com/pantsbuild/pants/pull/9965>`_
+  `PR #9906 <https://github.com/pantsbuild/pants/pull/9906>`_
+
+* Add support for returning multiple coverage reports for each MergedCoverageData. (#9992)
+  `Issue #9968
 <https://github.com/pantsbuild/pants/issues/9968
>`_
+  `PR #9992 <https://github.com/pantsbuild/pants/pull/9992>`_
+
+* Support for marking source roots via a specially-named file. (#9977)
+  `PR #9977 <https://github.com/pantsbuild/pants/pull/9977>`_
+
+* Expose v2 goals from RunTracker. (#9963)
+  `PR #9963 <https://github.com/pantsbuild/pants/pull/9963>`_
+
+* remoting: allow override of target_platform (#9954)
+  `PR #9954 <https://github.com/pantsbuild/pants/pull/9954>`_
+
+* remoting: add streaming remote execution client (#9897)
+  `PR #9897 <https://github.com/pantsbuild/pants/pull/9897>`_
+
+* Log timestamps should be in the local time zone (#9927)
+  `PR #9927 <https://github.com/pantsbuild/pants/pull/9927>`_
+
+* Attach stdout/stderr digests to workunits (#9906)
+  `PR #9906 <https://github.com/pantsbuild/pants/pull/9906>`_
+
+* Add a Python import parser for dependency inference (#9900)
+  `PR #9900 <https://github.com/pantsbuild/pants/pull/9900>`_
```

Comment:
Probably refactoring changes.

Reply:
Stdout/stderr digests is definitely a new feature.